utils/storybook/withSnapshotsEnabled.ts

```diff
@@ -0,0 +1,55 @@
+type StoryDetails = {
+  title: string;
+  component?: React.ReactNode;
+  decorators?: any[];
+  parameters?: {
+    [key: string]: any;
+  };
+};
+
+type StoryFn = {
+  (): JSX.Element;
+  story?: {
+    parameters?: {
+      chromatic?: {};
+    };
+  };
+};
+
+/**
+ * A function to enable chromatic snapshots for a single story or a set of stories.
+ * To apply to all stories in a file, wrap the default export of a CSF story file.
+ * To apply to a single story, wrap the story fn.
+ *
+ * TODO: Clean up after optional chaining is enabled
+ */
+export const withSnapshotsEnabled: {
```

Comment:
Note: I looked at making this a regular storybook decorator so you could do something like
```tsx
export default {
  title: 'My Story',
  decorators: [withSnapshotsEnabled]
}
```
However, editing the `storyFn.story` object doesn't seem to work from within the decorator function. This must be a limitation of the Storybook decorator API.